app/services/exports/case_log_export_service.rb

```diff
@@ -45,13 +45,20 @@ def retrieve_case_logs
 
     def build_export_string_io(case_logs)
       doc = Nokogiri::XML("<forms/>")
+      pattern_age = /age\d_known/
+      omitted_attrs = %w[ethnic_group]
 
       case_logs.each do |case_log|
         form = doc.create_element("form")
         doc.at("forms") << form
         case_log.attributes.each do |key, _|
-          value = case_log.read_attribute_before_type_cast(key)
-          form << doc.create_element(key, value)
+          # omit _known fileds from export
```

Comment:
Spelling on fields. Could also possibly wrap that in a method to avoid the need for a comment? Something like:

```
if omitted_field?(key)
```